### PR TITLE
Adding Material UI and next-i18next

### PR DESCRIPTION
**Pull request by ipr0310**

### CODE REVIEW

File: features/home/components/OfflineCounter.tsx [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
-import {actions} from '../slice';
-import {useAppDispatch, useAppSelector} from '../../../states/hooks';
+import { countSelector } from "../selectors";
+import { actions } from "../slice";
+import { useAppDispatch, useAppSelector } from "../../../states/hooks";
 
-export const OfflineCounter = () => {
-    const dispatch = useAppDispatch()
-    const count = useAppSelector(countSelector)
+// Material ui
```
Comment (signum-nft) [inline features/home/components/OfflineCounter.tsx:5]: useless comment! don't do that

File: features/home/index.tsx [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
-import {OfflineCounter} from './components/OfflineCounter';
-import {ChainStatus} from './components/ChainStatus';
+import type { NextPage } from "next";
+import Head from "next/head";
+import styles from "./Home.module.css";
+import { OfflineCounter } from "./components/OfflineCounter";
+import { ChainStatus } from "./components/ChainStatus";
 
-export const Home: NextPage = () => {
+// Material-ui
```
Comment (signum-nft) [inline features/home/index.tsx:7]: same!

File: features/home/index.tsx [hunk 3]
```diff
@@ -20,58 +21,17 @@ export const Home: NextPage = () => {
...
 
-        <OfflineCounter/>
+        <OfflineCounter />
 
-        <ChainStatus/>
+        <ChainStatus />
 
         <p className={styles.description}>
-          Get started by editing{' '}
+          Get started by editing{" "}
```
Comment (signum-nft) [inline features/home/index.tsx:29]: we need to check the code style.... to not mix up stuff....we have eslint running.... need to observe if this works consistently. I have setup husky for auto linting on commit. Should run on your side also

File: pages/_app.tsx [hunk 4]
```diff
@@ -1,23 +1,52 @@
...
+import { ThemeProvider } from "@mui/material/styles";
+import { CacheProvider, EmotionCache } from "@emotion/react/";
+import CssBaseline from "@mui/material/CssBaseline";
+import theme from "../styles/theme";
+import createEmotionCache from "../styles/createEmotionCache";
+
+// Redux persistor
+const persistor = persistStore(store);
+
+// Client-side cache, shared for the whole session of the user in the browser.
```
Comment (signum-nft) [inline pages/_app.tsx:22]: omit these useless comments

File: features/home/index.tsx [hunk 5]
```diff
@@ -20,58 +21,17 @@ export const Home: NextPage = () => {
...
-          </span>
-        </a>
-      </footer>
     </div>
-  )
-}
+  );
+};
+
+export default Home;
```
Comment (signum-nft) [inline features/home/index.tsx:37]: No default exports please!

**Review by signum-nft — CHANGES_REQUESTED** [review]
Please remove the comments, and make Home being returned by name and not default

File: .env.development [hunk 6]
```diff
@@ -1,2 +1,4 @@
 # add development environment specific overrides here
```
Comment (signum-nft) [inline .env.development:1]: move this this .env (make a copy)

File: app/components/SEOMetaTags/index.tsx [hunk 7]
```diff
@@ -0,0 +1,121 @@
+import Head from "next/head";
+
+export interface MetaTagsProps {
```
Comment (signum-nft) [inline app/components/SEOMetaTags/index.tsx:3]: Don't be too overzealous --- You are copying and pasting from the wild...Who is "Otekdo"? What is FB ID : 681600232333515 ? 

Be patient, young padawan

File: app/components/SEOMetaTags/index.tsx [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+
+      {/* Website */}
+      <script
+        type="application/ld+json"
+        dangerouslySetInnerHTML={{
+          __html: JSON.stringify({
+            "@context": "http://schema.org",
+            "@type": "WebSite",
+            alternateName: "Signumart",
+            url: "https://www.signumart.io/",
```
Comment (signum-nft) [inline app/components/SEOMetaTags/index.tsx:115]: I hate `www` - don't get infected by Frank... NO WWW

File: styles/theme.ts [hunk 9]
```diff
@@ -0,0 +1,19 @@
+import { createTheme } from "@mui/material/styles";
+import { red } from "@mui/material/colors";
+
+// Create a theme instance.
```
Comment (signum-nft) [inline styles/theme.ts:4]: really... I thought `createTheme` would cook a coffee for me :grin:

File: app/components/SEOMetaTags/index.tsx (hunk 7 above)
Comment (ipr0310) [reply app/components/SEOMetaTags/index.tsx:3]: It is in case this may implement FB pixel

**Review by signum-nft — APPROVED** [review]

**Pull request merged**